blockchain/network.py

```diff
@@ -463,6 +470,82 @@ def timed_receipt_request(self):
                 ver_ids = node.client.receipt_request(node.pass_phrase)
                 self.resolve_data(node, ver_ids, node.phases)  # node.phases may present problems since it's in binary
 
+    def subscription_feed(self):
+        """ request transactions with associated verification records from subscription """
+        subscriptions = subscribe_to_db.get_all()
+        for subscription in subscriptions:
+            subscription_node = self.get_subscription_node(subscription)
+            if subscription_node:
+                # sign subscription
+                self.processing_node.get_subscription_signature(subscription)
```

Comment:
If it's signed each time (every request) the signature timestamp should suffice. 

If you're thinking about a timestamp for the sub in the `sub_from` or `sub_to` Yeah, maybe they should have a create timestamp.